concent_api/core/tests/utils.py

```diff
@@ -233,13 +251,14 @@ def _get_deserialized_task_to_compute(
 
     def _get_deserialized_ack_report_computed_task(
         self,
-        timestamp       = None,
-        deadline        = None,
-        subtask_id      = '1',
-        report_computed_task = None,
-        task_to_compute = None,
-        sign_with_private_key = None,
-    ):
+        timestamp: Union[str, datetime.datetime, None] = None,
+        deadline: Union[str, int, None]=None,
```

Comment:
should be `Union[str, int, None] = None`

Reply:
done